docs/source/models.rst

```diff
@@ -352,6 +348,12 @@ by hand or :py:func:`inferred <mlflow.models.infer_signature>` from datasets wit
 (e.g. the training dataset with target column omitted) and valid model outputs (e.g. model
 predictions generated on the training dataset).
 
+You may also include a signature with your model by omitting the :py:class:`signature object
+<mlflow.models.ModelSignature>` from the log_model call and instead passing an :ref:`input example <input-example>`.
+Then, for supported MLflow flavors (``sklearn``, ``xgboost``, ``tensorflow``, ``transformers``, and
+``spark``), log_model will automatically infer the signature from the input example and the model's
```

Comment:
We only support these flavors?

Reply:
I'm building support for more flavors. I'll update this list as I implement and land more model flavors. This PR captures the flavors supported in https://github.com/mlflow/mlflow/pull/8725